app-editors/fte/fte-20110708-r1.ebuild

```diff
@@ -48,6 +52,7 @@ src_prepare() {
 	# epatch "${FILESDIR}"/${PN}-new_keyword.patch
 	# epatch "${FILESDIR}"/${PN}-slang.patch
 	# epatch "${FILESDIR}"/${PN}-interix.patch
+	default
 
 	[[ -e /usr/include/linux/keyboard.h ]] && \
 		sed /usr/include/linux/keyboard.h -e '/wait.h/d' > src/hacked_keyboard.h
```

Comment:
you're doing filesystem I/O here, hence you need `|| die`. You can't just add it though, because it will get subshell fangled-bla, hence turn this into a proper if; then